.NET/Microsoft.Recognizers.Text.DateTime/Spanish/Extractors/SpanishDateTimePeriodExtractorConfiguration.cs

```diff
@@ -6,7 +6,7 @@ namespace Microsoft.Recognizers.Text.DateTime.Spanish
 {
     public class SpanishDateTimePeriodExtractorConfiguration : IDateTimePeriodExtractorConfiguration
     {
-        public static readonly Regex NumberCombinedWithUnit = new Regex(DateTimeDefinitions.NumberCombinedWithUnit, RegexOptions.IgnoreCase | RegexOptions.Singleline);
+        public static readonly Regex NumberCombinedWithUnit = new Regex(DateTimeDefinitions.DateTimePeriod_NumberCombinedWithUnit, RegexOptions.IgnoreCase | RegexOptions.Singleline);

```

Comment:
Should we follow CamelCase naming?

Reply:
Not an issue. Let's keep the current form.